File: 782-jewels-and-stones/jewels-and-stones.js
/**
 * @param {string} jewels
 * @param {string} stones
 * @return {number}
 */
var numJewelsInStones = function(jewels, stones) {
    let j=new Set(jewels);
    let count=0;
    for(c of stones){
        if(j.has(c)==true){
            count++;
        }
    }
    return count;
};

File: 782-jewels-and-stones/jewels-and-stones.go
func numJewelsInStones(jewels string, stones string) int {
    var m =map[rune]bool{}
    count:=0
    for _,jewel:=range jewels{
        m[jewel]=true
    }
    for _,stone:=range stones{
        if m[stone]{
            count++
        }
    }
    return count
}